scripts/update_e2e_coverage.py

```diff
@@ -114,17 +158,11 @@
 MAIN_URL = 'https://github.com/google/iree/tree/main'
 TARGETS_URL = os.path.join(MAIN_URL, 'iree/compiler/Dialect/HAL/Target')
 
-E2E_COVERAGE_DESCRIPTION = f"""# TensorFlow End to End Coverage
-There are three backend [targets]({TARGETS_URL}) in IREE:
-
-- vmla
-- llvm-ir
-- vulkan-spirv
-
-The table shows the supported TensorFlow functions and models on each backend.
-It is auto-generated from IREE's test status.
-
-"""
+BACKEND_INFO = f"""IREE has three backend
+[targets]({TARGETS_URL}):
+`vmla`, `llvm-ir` and `vulkan-spirv`. We also test TFLite in our infrastructure
```

Comment:
We should clarify this "three backend targets" at some point.

These are our current compiler backends:
>  DYLIB-LLVM-AOT
>  LLVM-IR
>  Metal-SPIRV
>  Vulkan-SPIRV
>  VMLA

LLVM-IR and DYLIB-LLVM-AOT should have very similar, if not the same, coverage.
Metal-SPIRV and Vulkan-SPIRV share SPIR-V components but will differ in their runtime capabilities.

Reply:
What would you say the mapping is between `iree_vmla`, `iree_llvmjit`, `iree_vulkan` and the backends you listed? VMLA is pretty obviously 1:1, and I'm guessing that `iree_llvmjit` is `LLVM-IR` and `DYLIB-LLVM-AOT` is untested. Does `iree_vulkan` (compiler target `vulkan-*`) target `Vulkan-SPIRV` and `Metal-SPIRV` depending on the OS it's running in, or do we need to add another backend to the tf infra for MacOS?

I think this section should be updated by someone more familiar with IREE's mid-end, and we should probably standardize the names between the tf infra and IREE itself (e.g. `iree_vmla` -> `vmla`, `iree_llvmjit` -> `llvm-ir`, ...).